package.json

```diff
@@ -2,15 +2,25 @@
     "name": "ziggy",
     "version": "0.5.0",
     "description": "Generates a Blade directive exporting all of your named Laravel routes. Also provides a nice route() helper function in JavaScript.",
+    "keywords": ["laravel", "routes", "ziggy"],
     "main": "src/js/route.js",
     "directories": {
         "test": "tests/js"
     },
     "repository": {
         "type": "git",
-        "url": "git+https://github.com/tightenco/ziggy.git"
+        "url": "https://github.com/tightenco/ziggy.git"
     },
-    "author": "Daniel Coulbourne",
+    "authors": [
+        {
+            "name": "Daniel Coulbourne",
+            "email": "daniel@tighten.co"
+        },
+        {
+            "name": "Matt Stauffer",
```

Comment:
I don't need to be listed on author on a project where I did very little! :) But I'm fine for Jake and Jacob to be added, if y'all write much code.